legal-api/src/legal_api/reports/report.py

```diff
@@ -61,6 +62,15 @@ def _get_report_filename(self):
     def _get_filing_description(self):
         return self._get_primary_filing()['title']
 
+    def _format_directors(self, directors):
+        for director in directors:
+            try:
```

Comment:
As per the last discussion all formatting will be moved to template and I think this should move to the respective template